hassio/src/dialogs/registries/dialog-hassio-registries.ts

```diff
@@ -110,16 +110,15 @@ class HassioRegistriesDialog extends LitElement {
                             )}:
                             ${entry.username}</span
                           >
-                          <mwc-icon-button
+                          <ha-icon-button
```

Comment:
Don't forget to also update the imports

Reply:
In the end all imports except on in `ha-icon-button.ts` should be obsolete. But I'll put it on the todo list.

Reply:
Yes, but I think a lot now only import `mwc-icon-button` and `ha-svg-icon`, those should be changed in `ha-icon-button`

Reply:
True, but usually VSCode tells me if that is the case and I add the import. In the end the GitHub actions will tell me once I move out of draft 😄

Reply:
Unfortunately, it doesn't tell us that... :-(

Reply:
Imports updated ... hopefully I did not miss "any" ... well realistically "not too many".